packages/core/template/default/index.md

```diff
@@ -1,132 +1,75 @@
-</box>
-<box type="important" dismissible header="Box heading">
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-</box>
+---
 
-<br>
+##### **User Guide: Working with Sites**
 
-# Heading 3
+> Learn how to modify site properties, apply themes, and enable/disable plugins for your MarkBind site.
 
-<panel header="Expandable panel" type="info">
-  Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
-</panel>
-<br>
-<panel header="Expanded panel on load" type="info" expanded>
-  ...
-</panel>
-<br>
-<panel header="Expanded panel header" alt="Minimized panel header" type="success" minimized>
-  ...
-</panel>
-<br>
-<panel header="Expanded panel header" alt="Minimized panel header" type="seamless">
-  ...
-</panel>
-<br>
+More info in: _<a href="https://markbind.org/userGuide/workingWithSites.html" target="_blank">User Guide → Working with Sites</a>_
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-<panel header="___Minimal panel **->**___" type="minimal" alt="Minimal panel" popup-url="https://markbind.org/userGuide/components/presentation.html#panels" no-switch>
-Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-</panel>
+---
+
+##### **User Guide: Full Syntax Reference**
 
-Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+> Refer to our Full Syntax Reference page to find a specific feature or component that you want to use in your MarkBind site.
+
+More info in: _<a href="https://markbind.org/userGuide/fullSyntaxReference.html" target="_blank">User Guide → Full Syntax Reference</a>_
+
+</panel>
```

Comment:
Can we add some line breaks after the panel here so that the panel isn't tight on the bottom of the page?
<img width="751" alt="Screenshot 2023-03-29 at 4 51 45 PM" src="https://user-images.githubusercontent.com/61113575/228486202-6cce6cf0-77e1-4a43-8269-45d3e625bdec.png">

Reply:
Thanks for the suggestion! Updated in [eface18](https://github.com/MarkBind/markbind/pull/2225/commits/eface184bfc22fc7a9db7f55e82882f47f018d5e).